Makefile

```diff
@@ -33,11 +32,15 @@ endif
 
 ##@ Dependencies
 .PHONY: install-all-deps
-install-all-deps: install-build-deps install-site-deps ## Install all dependencies for building both binary and user docs)
+install-all-deps: install-site-deps ## Install all dependencies for building both binary and user docs)
 
-.PHONY: install-build-deps
-install-build-deps: ## Install dependencies (packages and tools)
-	build/scripts/install-build-deps.sh
+.PHONY: install-tools
+install-tools: ## Install dependencies for code generation and test execution
```

Comment:
Is there a reason for renaming this target? There are other references to the old target name `install-build-deps` in various other places that also need to be updated.

Reply:
its a bit mis-leading, the aren't just build deps but also deps required for testing (cert generation), and this matched the `tools.go` a bit better. I can go either way though - let me know which way we want go and I'll make sure any references are updated

Reply:
reverted